usvg/src/convert/text/convert.rs

```diff
@@ -317,6 +317,14 @@ fn resolve_text_flow(
         node.resolve_length(AId::StartOffset, state, 0.0)
     };
 
+    let path = if let Some(node_transform) = path_node.attribute::<Transform>(AId::Transform) {
```

Comment:
Initially, I though that you are making an unnecessary cloning, but since the same path can be referenced by multiple textPath's, your version is actually correct.

Also, could you move it before `start_offset`?

Reply:
Rust wouldn't even let me mutate the `Rc`, so I didn't have a choice. The copy on write should work out nicely I think.